### PR TITLE
Fixing ManualTrigger input (#602)

### PR DESCRIPTION
**Pull request by mathewc**

### CODE REVIEW

File: src/WebJobs.Script/Binding/ManualBinding.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+{
+    public class ManualBinding : FunctionBinding
+    {
+        public ManualBinding(ScriptHostConfiguration config, BindingMetadata metadata, FileAccess access) : base (config, metadata, access)
+        {
+        }
+
+        public override Task BindAsync(BindingContext context)
+        {
+            throw new NotImplementedException();
```
Comment (mathewc) [inline src/WebJobs.Script/Binding/ManualBinding.cs]: This binding is only used for trigger input, so has no BindAsync impl. Really the whole binding is just a noop, and inherits its behavior from base.

File: test/WebJobs.Script.Tests/EndToEndTestsBase.cs [hunk 2]
```diff
@@ -112,6 +112,28 @@ public EndToEndTestsBase(TTestFixture fixture)
         }
 
         [Fact]
+        public async Task ManualTrigger_Invoke_Succeeds()
```
Comment (mathewc) [inline test/WebJobs.Script.Tests/EndToEndTestsBase.cs]: I added a single e2e test here, which runs for ALL the languages

File: test/WebJobs.Script.Tests/NodeEndToEndTests.cs [hunk 3]
```diff
@@ -187,24 +187,6 @@ public void ErrorFunction_DoesNotBringDownHost()
         }
 
         [Fact]
-        public async Task ManualTrigger()
```
Comment (mathewc) [inline test/WebJobs.Script.Tests/NodeEndToEndTests.cs]: Pushed the stuff this guy was testing related to logging into its own scenario test

File: test/WebJobs.Script.Tests/PowerShellFunctionGenerationTests.cs [hunk 4]
```diff
@@ -15,7 +15,7 @@
 using Newtonsoft.Json.Linq;
 using Xunit;
 
-namespace Microsoft.Azure.WebJobs.Script.Tests.TestScripts
```
Comment (mathewc) [inline test/WebJobs.Script.Tests/PowerShellFunctionGenerationTests.cs]: This file was in the wrong folder (TestScripts) so I moved it

**Comment by brettsam** [comment]
:shipit:

**Pull request merged**